packages/cli/src/command-implementations/project-command.ts

```diff
@@ -80,24 +78,6 @@ export abstract class ProjectCommand<C extends ProjectCommandDefinition> extends
     }
   }
 
-  protected prepareIntegrations(botImpl: BotImpl, botInfo: bpclient.Bot): IntegrationInstances {
```

Comment:
there is no reason why this was not using the `setNullOnMissingValues` functions from `./utils/records`